public/static/cql/upenn_her2_library.cql

```diff
@@ -39,12 +39,16 @@ code "Abnormal estradiol (finding) code": '131064004' from "SNOMEDCT" display 'A
 
 // SNOMED Procedures
 code "Chemotherapy (procedure) code": '367336001' from "SNOMEDCT" display 'Chemotherapy (procedure)'
+code "Lumpectomy of breast (procedure) code": '392021009' from "SNOMEDCT" display 'Lumpectomy of breast (procedure)'
 code "Oophorectomy (procedure) code": '83152002' from "SNOMEDCT" display 'Oophorectomy (procedure)'
 
 
 // RXNORM Medication Request
+code "adotrastuzumab emtansine code": '1371041' from "RXNORM" display 'ado-trastuzumab emtansine'
 code "trastuzumab code": '224905' from "RXNORM" display 'trastuzumab'
 code "anastrozole code": '84857' from "RXNORM" display 'anastrozole'
+code "pertuzumab code": '1298944' from "RXNORM" display 'pertuzumab'
```

Comment:
I'm not sure what any of those acronyms mean... but the codes were all selected from RxNav. I searched the drug name and took the code listed. Are you suggesting to select one of the specific items instead of just the generic drug code?